bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ua-parser-js",
-  "version": "0.7.28",
+  "version": "0.7.31",
```

Comment:
why `0.7.31`?

Reply:
The latest version from ua parser is `0.7.31`, so when I pull the code and merge it, it becomes this version number.